SimpleTests/MeshTests/MPI/NodePartitionedMeshTester.cpp

```diff
@@ -0,0 +1,120 @@
+/*!
+  \file NodePartitionedMeshTester.cpp
+  \author Wenqing Wang
+  \date   2014.11
+  \brief  Test class readNodePartitionedMesh to read node-wise partitioned mesh with MPI functions.
+
+  \copyright
+  Copyright (c) 2012-2014, OpenGeoSys Community (http://www.opengeosys.org)
+             Distributed under a Modified BSD License.
+               See accompanying file LICENSE.txt or
+               http://www.opengeosys.org/project/license
+*/
+
+#include <iomanip>
+#include <fstream>
+#include <string>
+
+#ifdef USE_MPI
```

Comment:
i think all `#ifdef USE_MPI` in this file is unnecessary since this file is compiled only with MPI.